docs/core/tools/csproj.md

```diff
@@ -264,3 +264,37 @@ Base path for the *.nuspec* file.
 
 ### NuspecProperties
 Semicolon separated list of key=value pairs.
+
+## AssemblyInfo properties
+[Assembly attributes](../../framework/app-domains/set-assembly-attributes.md) that were typically present in an *AssemblyInfo* file are now automatically generated from properties.
+
+### Properties per attribute
+
+Each attribute has a property that control it's content and another to disable it's generation as follow :
+
+| Attribute                                                      | Property               | Property to disable                             |
+|----------------------------------------------------------------|------------------------|-------------------------------------------------|
+| <xref:System.Reflection.AssemblyCompanyAttribute>              | `Company`              | `GenerateAssemblyCompanyAttribute`              |
+| <xref:System.Reflection.AssemblyConfigurationAttribute>        | `Configuration`        | `GenerateAssemblyConfigurationAttribute`        |
+| <xref:System.Reflection.AssemblyCopyrightAttribute>            | `Copyright`            | `GenerateAssemblyCopyrightAttribute`            |
+| <xref:System.Reflection.AssemblyDescriptionAttribute>          | `Description`          | `GenerateAssemblyDescriptionAttribute`          |
+| <xref:System.Reflection.AssemblyFileVersionAttribute>          | `FileVersion`          | `GenerateAssemblyFileVersionAttribute`          |
+| <xref:System.Reflection.AssemblyInformationalVersionAttribute> | `InformationalVersion` | `GenerateAssemblyInformationalVersionAttribute` |
+| <xref:System.Reflection.AssemblyProductAttribute>              | `Product`              | `GenerateAssemblyProductAttribute`              |
+| <xref:System.Reflection.AssemblyTitleAttribute>                | `AssemblyTitle`        | `GenerateAssemblyTitleAttribute`                |
+| <xref:System.Reflection.AssemblyVersionAttribute>              | `AssemblyVersion`      | `GenerateAssemblyVersionAttribute`              |
+| <xref:System.Resources.NeutralResourcesLanguageAttribute>      | `NeutralLanguage`      | `GenerateNeutralResourcesLanguageAttribute`     |
+
+Notes:
+
+* `AssemblyVersion` and `FileVersion` default is to take the value of `$(Version)` without suffix. For example if `$(Version)` is `1.2.3-beta.4` then the value would be `1.2.3`.
+* `InformationalVersion` defaults to the value of `$(Version)`.
+* `InformationalVersion` has `$(SourceRevisionId)` apended if the property is present (Can be disabled using `IncludeSourceRevisionInInformationalVersion`).
```

Comment:
apended  -> appended 

I'd remove the last sentence from parentheses and add a subject:
It can be disabled using `IncludeSourceRevisionInInformationalVersion`.